documentation/asciidoc/accessories/iqaudio/faq.adoc

```diff
@@ -53,17 +51,17 @@ audio_output {
 
 === Hardware information
 
-* PCB screws are all m2.5
-* PCB standoffs (for CASE) are 5mm male / female
-* PCB standoffs (for Raspberry Pi to audio products) are 9mm female / female
-* PCB standoffs (for XLR to DAC PRO) are 8mm female / male
-* PCB standoffs (for the official Raspberry Pi 7-inch display) are 5mm male / female
-* The rotary encoders we have used and tested are the ALPHA 3 pin Rotary Encoder
-RE160F-40E3-20A-24P, the ALPS EC12E2430804 (RS: 729-5848), the Bourns ECW0JB24-AC0006L (RS: 263-2839)
-* The Barrel connector used for powering the DigiAMP+ is 2.5mmID, 5.5mmOD, 11mm
-* The DigiAMP{plus} is designed to operate with a 12v 3amp to 24v supply such as the XPPower
-VEC65US19 or similar
-* The DigiAMP{plus} uses CamdenBoss two part connectors. Those fitted to the PCB are:
-CTBP9350/2AO
-* The speaker terminal used on the Codec Zero will accept wires of between 14 ~ 26 AWG
-(wire of max 1.6mm in diameter)
\ No newline at end of file
+* PCB screws are all m2.5.
+* PCB standoffs (for case) are 5mm male/female.
+* PCB standoffs (for Raspberry Pi to audio products) are 9mm female/female.
+* PCB standoffs (for XLR to DAC PRO) are 8mm female/male.
+* PCB standoffs (for the official Raspberry Pi 7-inch display) are 5mm male/female.
+* The rotary encoders we have used and tested are the Alpha three-pin rotary encoder
+RE160F-40E3-20A-24P, the ALPS EC12E2430804 (RS: 729-5848), and the Bourns ECW0JB24-AC0006L (RS: 263-2839).
+* The barrel connector used for powering the DigiAMP{plus} is 2.5mmID, 5.5mmOD, 11mm.
```

Comment:
Space between 'mm' and 'ID' and 'OD'?

Reply:
cf. https://www.raspberrypi.com/documentation/accessories/build-hat.html#powering-the-build-hat

Reply:
We should describe it (in both places) like this then, "via a DC 5521 centre positive barrel connector (5.5mm × 2.1mm × 11mm)" assuming it is a centre positive barrel?

Reply:
...except the connector here has a 2.5mm inside diameter, which is different to the Build HAT's 2.1mm inside diameter :stuck_out_tongue_winking_eye:

Reply:
...and this is why we all hate barrel jacks

Reply:
This one also requires a conversation in person. Let's talk tomorrow.